src/scenes/Species/SpeciesListView.tsx

```diff
@@ -278,7 +278,6 @@ export default function SpeciesListView({ reloadData, species }: SpeciesListProp
         ? [
             { name: 'growthForm', label: strings.GROWTH_FORM, type: 'multiple_selection' },
             { name: 'conservationCategory', label: strings.CONSERVATION_CATEGORY, type: 'multiple_selection' },
-            { name: 'rare', label: strings.RARE, type: 'multiple_selection' },
```

Comment:
Oops, I should have clarified, we need this table column.
I meant removing `rare` from the list of fields on new line # 295. We don't use the values of `rare` from the response.